tests/reference/run_dbg-test_assert_01-2f34744.stderr

```diff
@@ -1,5 +1,2 @@
-  File "tests/runtime_errors/test_assert_01.py", line 4
-    test()
-  File "tests/runtime_errors/test_assert_01.py", line 2
-    assert False
+warning: The `runtime stacktrace` is not enabled. To get the stacktraces, re-build LPython with `-DWITH_RUNTIME_STACKTRACE=yes`
```

Comment:
Use `--skip-run-with-dbg`, while you update tests, so do `python run_tests.py -u --skip-run-with-dbg`. I forgot to document this in the README.

Reply:
<img width="442" alt="image" src="https://user-images.githubusercontent.com/90216905/218377555-c3ebf9e5-05ee-4e5f-b26e-4535934e655d.png">

Reply:
It isn't changing anything locally though.

Reply:
Yup. This is correct as our tests do not test the above changes.

Reply:
How can I correct the failing ubuntu and macos tests ? Manually changing back the `run-with-dbg` files ?

Reply:
Undo the updated references and force push here.

Reply:
My apologies for an incorrect attempt at reverting the initial commit. Now I have undone the updated references successfully.